mozilla-services/socorro
# bug 1602120: sqs local dev

This updates localstack image to 0.10.5 which is the latest one. It also switches the localstack-s3 service port from 5000 to 4572 which is the default one used by that image.

This also adds:
    
* configuration for `sqs_endpoint_url`
* `sqs_cli` script
* changes `resource.boto.endpoint_url` to `resource.boto.s3_endpoint_url` to differentiate it with the `sqs_endpoint_url`
* adds localstack-sqs service and relevant startup bits
* moves click to a default dependency since we use it in `sqs_cli`
* fixes some weird configuration things in `docker/run_tests.sh`
* removes some unused env vars

To test:

1. do `make build`
2. do `make runservices` -- this should start the localstack-sqs service
2. do `make test` -- this should run the tests with the fixed configuration
